dio/lib/src/dio_mixin.dart

```diff
@@ -447,10 +444,10 @@ abstract class DioMixin implements Dio {
     FutureOr<dynamic> Function(dynamic, StackTrace) errorInterceptorWrapper(
```

Comment:
The stack trace parameter is no longer needed. And the error type can be promoted to `Object`.

```suggestion
    FutureOr<dynamic> Function(Object) errorInterceptorWrapper(
```

Reply:
No, this doesn't work, there are dynamic calls after this..

Reply:
Oh right, but the stack trace should be available to remove.

Reply:
I refactored this a bit, so now it works.